webpack-demo/src/addRemove.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars */
+// eslint-disable-next-line import/no-cycle
+import { generateList, saveDataLocally } from './index.js';
+
+export const add = () => {
+  const textInputValue = document.querySelector('#add-input').value;
+  const retrievedList = JSON.parse(localStorage.getItem('list'));
+  const TaskItem = {
+    description: textInputValue,
+    completed: false,
+    index: retrievedList.length + 1,
+  };
+
+  const addedItem = TaskItem;
+  retrievedList.push(addedItem);
+  saveDataLocally(retrievedList);
+};
```

Comment:
- [ ] <b>REQUIRED:</b> `Add` Functionality is not working. Kindly address the issue and fix it.
![Screen Shot 2022-01-15 at 04 22 22](https://user-images.githubusercontent.com/45217364/149605438-db3aa2b4-0a7c-44ab-af6e-7eda1f572a41.png)

Reply:
Thank you for the comment and suggestion. After moving my files from webpack-demo/ to the root directory, I tested it and my console throws no error when adding, removing or editing elements.

Reply:
** This change generated linter errors with lodash and webpack imports that I was not able to correct. After reverting, I myself am not getting any errors when running my code for add, remove or edit using npm start. (???)  Unfortunately, I needed to generate a new branch to revert the changes back to keep the directory.